KerbTrack/KerbTrack.cs

```diff
@@ -25,7 +25,8 @@ public class KerbTrack : MonoBehaviour
         public static ITracker tracker;
 
         // [...]GameData\KerbTrack\Plugins\PluginData\KerbTrack\settings.cfg
-        private string savePath = AssemblyLoader.loadedAssemblies.GetPathByType(typeof(KerbTrack)).Replace("/", @"\\") + @"\\settings.cfg";
+        private string savePath = System.IO.Path.Combine(
+            AssemblyLoader.loadedAssemblies.GetPathByType(typeof(KerbTrack)), "settings.cfg");
```

Comment:
This finds the settings file in a platform agnostic way, should work on all operating systems. The original code was doing very funky things on linux.